modules/DisplayDateTime.js

```diff
@@ -0,0 +1,8 @@
+import { DateTime } from 'luxon';
+
+export default () => {
+  const dateContainer = document.querySelector('#date-container');
+  dateContainer.innerHTML = '';
+  const date = DateTime.now().toFormat('FF');
+  dateContainer.append(date);
+};
```

Comment:
- [ ] Good job refactoring your code to use arrow functions. However, the date is still not being shown on the screen and if I try to add a new book I cannot because the app does nothing when I click on the `Add New Book` button. Please check and apply the necessary changes to fix this issue.

Reply:
Hello, this is very strange as it works fine for me when I run npx parcel index.html

Reply:
Yes, it worked now. It was my fault because I was running the app using live server and not parcel. I am going to add a new review to GitHub approving your project, but you will need to ask for a new review in your dashboard in order to get the submit button. In this case, please explain to the new reviewer about my mistake, and tell him to check the PR so he can just approve the project. You can do this by using the `Additional information` option before asking for a review.

Sorry for the inconvenience and good job!

Reply:
Thanks!